_conferences/droidcon-ph-2018.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Droidcon"
+website: https://ph.droidcon.com/
+location: Manila, PH
```

Comment:
Is "PH" a common abbreviation? I had no idea what it meant.

Reply:
It means "Philippines". Will update it to avoid confusion.

Reply:
Yeah I had googled it. I just didn't know if it was common enough or it was just me. But thanks!